### PR TITLE
Add Badge Component

### PR DESCRIPTION
**Pull request by manojkumar8400**
Added Badges which includes:

1. Badges on text
2. Badges on Button
3. Badges on Icon
4. Badges on Avatar

### CODE REVIEW

File: documentation/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    }
+    .footer-btn {
+        font-size: 1.4rem;
+    }
+
+}
+
+.side-badge-btn {
+    background-color: var(--header);
+    border-radius: 25px;
```
Comment (Varu98) [inline documentation/badge/badge.css:32]: maybe try using rems for the border radius

File: documentation/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+.dot-badge {
+    background-color: green;
+    height: 20px;
+    width: 20px;
+    border-radius: 50%;
+    position: absolute;
+    bottom: 18px;
+    right: 6px;
+    /* bottom: -40px; */
+    /* left: 515px; */
```
Comment (Varu98) [inline documentation/badge/badge.css]: these comments might not be needed

**Review by Varu98 — APPROVED** [review]
Rest looks fine! 👍🏻

File: documentation/badge/badge.css (hunk 1 above)
Comment (manojkumar8400) [reply documentation/badge/badge.css:32]: I will take care in the future.

File: documentation/badge/badge.css (hunk 2 above)
Comment (manojkumar8400) [reply documentation/badge/badge.css]: It's my mistake I was totally forgot to remove it

**Review by gopalbbharadva — APPROVED** [review]

File: documentation/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    position: absolute;
+    bottom: 22px;
+    left: 20px;
+}
+
+.avtr-cont{
+    position: relative;
+}
+.onln-dot-badge-avtr {
+    width: 110px;
```
Comment (KoushikKintali) [inline documentation/badge/badge.css:95]: it is advised to use px, if the value is less than 10 otherwise rems or rems

**Review by KoushikKintali — APPROVED** [review]

File: documentation/badge/badge.css (hunk 3 above)
Comment (manojkumar8400) [reply documentation/badge/badge.css:95]: Ok I will do that.

**Comment by preethasri** [comment]
looks good

**Pull request merged**